src/controllers/bespokeOrStandardRules.controller.js

```diff
@@ -28,9 +31,12 @@ module.exports = class BespokeOrStandardRulesController extends BaseController {
       return this.doGet(request, h, errors)
     }
 
-    const isBespoke = request.payload['permit-type'] === 'bespoke'
+    const context = await RecoveryService.createApplicationContext(h)
+    // Save the permit type in the Data store
+    const permitType = request.payload['permit-type']
+    await DataStore.save(context, { permitType })
```

Comment:
We should validate the value of the permit-type parameter before saving it.

Reply:
Done